Update README with standardized config options (#4422)
DataDog/integrations-core: php_fpm/README.md

  - status_url: http://localhost/status # or whatever pm.status_path is set to in your PHP INI
    ping_url: http://localhost/ping     # or whatever ping.path is set to in your PHP INI
    ping_reply: pong                    # the reply to expect from ping; default is 'pong'

>  #  username: <YOUR_USERNAME>     # if the status and ping URLs require HTTP basic auth

Reviewer: ```suggestion
 #  username: <YOUR_USERNAME> # if the status and ping URLs require HTTP basic auth
```

* `ping_url` (Required) - URL for the PHP FPM ping page defined in the fpm pool config file (ping.path)
* `use_fastcgi` (Optional) - Communicate directly with PHP-FPM using FastCGI
* `ping_reply` (Required) - Reply from the ping_url. Unless you define a reply, it is `pong`

> * `username` (Optional) - Used if you have set basic authentication on the status and ping pages

Reviewer 2: ```suggestion
* `username` (Optional) - Used if basic authentication is set on the status and ping pages
```
